forem/forem
# Update to yarn latest

## What type of PR is this? (check all applicable)
- [x] Feature/DX

## Description
I've decided to back peddle my `pnpm` effort and focus on using latest `yarn`, because that's easier than forking a Heroku buildpack to get `pnpm` to work.

I may have misjudged Yarn Berry initially, and I think even without moving to `pnp`  it work just as well as `pnpm`.
- [x] tested on heroku canary
- [x] tested on forem fleet

Running `yarn dedupe` has also significantly cut down the node_module size.
## Related Tickets & Documents
No ticket was created for this

## QA Instructions, Screenshots, Recordings
- If you try `yarn install` locally, you'll find that `node_module` size is only 700MB
- Everything should work as usually
- All checks should be green.

## Added tests?

## Added to documentation?

## [optional] Are there any post deployment tasks we need to perform?
- [ ] Remove certain deprecated yarn vars
